bootstrap/013_warehouse_pools.sql

```diff
@@ -0,0 +1,43 @@
+
+CREATE OR REPLACE PROCEDURE ADMIN.CREATE_WAREHOUSE_POOL(name text, max_concurrent_credits int, allowed_roles array, snowflake_tag object, target_label text)
+    returns text
+    language python
+    runtime_version = "3.10"
+    handler = 'create_warehouse_pool'
+    packages = ('snowflake-snowpark-python', 'pydantic==1.*', 'snowflake-telemetry-python')
+    imports = ('{{stage}}/python/crud.zip', '{{stage}}/python/ml.zip')
+    EXECUTE AS OWNER
+AS
+$$
+from ml.warehouse_pool import compute_set_of_warehouses
+from crud.base import transaction
+from crud.warehouse_pools import Warehouse, WarehousePools
+def create_warehouse_pool(bare_session, pool_name: str, max_concurrent_credits: int, allowed_roles: list[str], snowflake_tag: dict, target_label: str):
+    warehouses = []
+    dataframe = compute_set_of_warehouses('Medium', max_concurrent_credits)
+    # TODO: get random number, secrets/random package wasn't working
+    rand_prefix = 123458
```

Comment:
if this is not working, should we move the logic to a python fn ? or, compute_set_of_warehouses() ?

Reply:
fixed it. defined a function and called it